### PR TITLE
test: add tests to improve code coverage for logging

### PR DESCRIPTION
**Pull request by emeeker2002**

### CODE REVIEW

**Comment by emeeker2002** [comment]
Got the logger test working with all checks passed except for the Console API Tests / build (pull_request) (Won't work right for now)

**Review by rsdmike — CHANGES_REQUESTED** [review]
Looking at this I think we've got some other issues at work here. Namely, the logging level doesn't seem to be working correctly. Looks to be an issue from the template we used. I know this is a code coverage task, but lets take a stab at using the interface provided in this issue: https://github.com/evrone/go-clean-template/issues/148 . We shouldnt need to have if conditions in our code as that would sort of eliminate the job of what the logging library should do. Can we take a stab at copying and pasting that and updating the code to use it?

File: pkg/logger/logger_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var mu sync.Mutex
+
+type loggerTest struct {
+	name         string
+	logLevel     string
+	testFunction func(t *testing.T, log *Logger, buf *bytes.Buffer)
```
Comment (matt-primrose) [inline pkg/logger/logger_test.go:22]: hey @emeeker2002 typically you'd want to have an expected result in your test struct. and then in each test case, you'd set the expected result.  That way all assets can be handled in the for loop at the end.

**Review by matt-primrose — CHANGES_REQUESTED** [review]
Update test struct and assert handling based on feedback in comment

File: pkg/logger/logger_test.go (hunk 1 above)
Comment (rsdmike) [reply pkg/logger/logger_test.go:22]: This would explain why the linter was wanting t.Helper in each function  which was a bit odd to me.

**Comment by rsdmike** [comment]
closed in place of #249 due to @emeeker2002 on vacation.

**Pull request closed without merging**